README.md

```diff
@@ -1,7 +1,17 @@
-<img src="img/AAA_long_plain.svg" width="75%" alt="AwesomeAndroidAlternatives">
-
-## This is the list of THE BEST apps according to me. You can prove me wrong on the [contributing page](CONTRIBUTING.md).
+<p align="center">
```

Comment:
I know about this `<p align="center">` trick, however I disliked it, because the font is left aligned, so please remove this `<p></p>` tags.